### PR TITLE
added solution method

### PR DESCRIPTION
**Pull request by tsehelnyk**

### CODE REVIEW

File: src/main/java/core/basesyntax/Palindrome.java [hunk 1]
```diff
@@ -20,7 +20,10 @@
      * <p>Пример: Madam, I'm Adam!</p>
      * <p>Результат: true</p>
      */
+
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.toLowerCase().replaceAll("[/\\W+/g]+", "");
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Palindrome.java]:
```suggestion
        text = text.toLowerCase().replaceAll("[\\W]", "");
```

File: src/main/java/core/basesyntax/Palindrome.java [hunk 2]
```diff
@@ -20,7 +20,10 @@
...
+        StringBuilder builder = new StringBuilder(text);
+        return text.equals(String.valueOf(builder.reverse()));
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Palindrome.java]:
```suggestion
        return text.equals(builder.reverse()toString());
```

Comment (tsehelnyk) [reply src/main/java/core/basesyntax/Palindrome.java]: What's different?

Comment (Sofasmile) [reply src/main/java/core/basesyntax/Palindrome.java]: String.valueOf return null if builder = null
toString() - throws NPE

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**